src/dishka/entities/provides_marker.py

```diff
@@ -1,15 +1,20 @@
-from typing import TYPE_CHECKING
+from __future__ import annotations
+
+from collections.abc import Sequence
+from typing import TYPE_CHECKING, Any
+
+__all__ = ["AnyOf"]
```

Comment:
do we really need it?
why `ProvideMultiple` in not here? it is imported in neighbor modules